docker/entrypoint.sh

```
@@ -68,15 +80,25 @@ run_sidecar() {
-    exec python3 /app/sidecar.py \
-        --port "$PORT" \
-        --bind "$BIND" \
-        --multiproc-dir "$MULTIPROC_DIR" \
-        --update-interval "$UPDATE_INTERVAL"
+    if [ "${REDIS_ENABLED:-false}" = "false" ]; then
+        echo "DEBUG: Executing python3 /app/sidecar.py with multiproc-dir. Arguments: sidecar --port \"$PORT\" --bind \"$BIND\" --multiproc-dir \"$MULTIPROC_DIR\" --update-interval \"$UPDATE_INTERVAL\" $@"
+        exec python3 /app/sidecar.py sidecar \
+            --port "$PORT" \
+            --bind "$BIND" \
+            --multiproc-dir "$MULTIPROC_DIR" \
+            --update-interval "$UPDATE_INTERVAL" $@ # Pass additional arguments
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)